RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java

```diff
@@ -266,37 +269,34 @@ private void popuplateDeletedDirectory(SleuthkitCase skCase, AbstractFile parent
     }
 
     /**
-     * Parse the $I file.
+     * Parse the $I file.  This file contains metadata information about deleted files
      *
      * File format prior to Windows 10:
-     * <table>
-     * <tr><th>Offset</th><th>Size</th><th>Description</th></tr>
-     * <tr><td>0</td><td>8</td><td>Header</td></tr>
-     * <tr><td>8</td><td>8</td><td>File Size</td></tr>
-     * <tr><td>16</td><td>8</td><td>Deleted Timestamp</td></tr>
-     * <tr><td>24</td><td>520</td><td>File Name</td></tr>
-     * </table>
-     *
+     * Offset  Size  Description
+     *   0       8     Header
+     *   8       8     File Size
+     *  16       8     Deleted Timestamp
+     *  24     520     File Name
+     * 
      * File format Windows 10+
-     * <table>
-     * <tr><th>Offset</th><th>Size</th><th>Description</th></tr>
-     * <tr><td>0</td><td>8</td><td>Header</td></tr>
-     * <tr><td>8</td><td>8</td><td>File Size</td></tr>
-     * <tr><td>16</td><td>8</td><td>Deleted TimeStamp</td></tr>
-     * <tr><td>24</td><td>4</td><td>File Name Length</td></tr>
-     * <tr><td>28</td><td>var</td><td>File Name</td></tr>
-     * </table>
-     *
+     * Offset  Size  Description
+     *   0       8     Header
+     *   8       8     File Size
+     *  16       8     Deleted TimeStamp
+     *  24       4     File Name Length
+     *  28     var     File Name
+     * 
```

Comment:
Why remove the formatting? Did this not build /format properly in doxygen? Does it build /format properly now?

Reply:
I will check it out.

Reply:
Install Doxygen.
http://www.doxygen.nl/download.html
Download and run the Windows installer. 

Run doxygen for Sleuthkit in ~sleuthkit/bindings/java/doxygen, then run doxygen for Autopsy in ~autopsy/docs/doxygen, ~autopsy/docs/doxygen-user, and ~autopsy/docs/doxygen-dev

Reply:
From what I see that table never shows up in the docs.  I also checked online in the 4.13 docs and I do not see it as well.

Reply:
Actually, that makes sense - this is not a public class. I'll accept the PR as is.